python/pyarrow/tests/test_pandas.py

```diff
@@ -4179,20 +4258,20 @@ def test_to_pandas_extension_dtypes_mapping():
     assert isinstance(result['a'].dtype, pd.PeriodDtype)
 
 
-def test_array_to_pandas():
+@pytest.mark.parametrize("arr",
+                         [pd.period_range("2012-01-01", periods=3, freq="D").array,
+                          pd.interval_range(1, 4).array])
```

Comment:
@jorisvandenbossche @danepitkin We can't use `pd` here because pandas may not be available.
This causes an error on "no pandas" environment: https://github.com/apache/arrow/actions/runs/5496447565/jobs/10016477233
This PR's CI succeeded because our "Without Pandas" job installed pandas implicitly. It has been fixed by #36542.

Could you open an issue for this and fix this?

Reply:
Fixed -> https://github.com/apache/arrow/pull/36586